src/modules/home/recommended-templates/RecommendedTemplates.tsx

```diff
@@ -7,15 +9,25 @@ import { templatesMock } from 'src/mocks/templates';
 import csx from './RecommendedTemplates.scss';
 
 export const RecommendedTemplates = () => {
+  const { push, location } = useHistory();
+
+  const goToDetails = useCallback(
+    (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      const templateId = e.currentTarget.getAttribute('data-id');
+      push(`app/templates/all/${templateId}`);
+    },
+    [location]
+  );
+
+  const recommendedTemplates = templatesMock.slice(0, 4);
```

Comment:
just templates without recommended will be better (imo) - variable names should be short - we already know its recommended templates because they are inside Recommented Templates component